bootstrap/bootstrap-api/src/main/java/com/alipay/sofa/rpc/bootstrap/DefaultProviderBootstrap.java

```diff
@@ -185,6 +187,12 @@ private void doExport() {
                 }
             }
 
+            //如果是泛型接口则需要在JSON序列化器中注册跟真实实现类的对应关系，因为反序列化时无法从泛型接口中拿到真实的数据类型
+            if (providerConfig.getProxyClass().getTypeParameters().length > 0) {
+                String serviceName = ConfigUniqueNameGenerator.getUniqueName(providerConfig);
+                JacksonSerializer.registerGenericService(serviceName, providerConfig.getRef().getClass().getName());
```

Comment:
We can treat GenericService as a general service. It may be not for JacksonSerializer only. How about move it to core module ?

Reply:
fixed